Source/WebCore/layout/formattingContexts/inline/InlineFormattingContext.h

```diff
@@ -79,6 +79,8 @@ class InlineFormattingContext {
     const BoxGeometry& geometryForBox(const Box&, std::optional<EscapeReason> = std::nullopt) const;
     BoxGeometry& geometryForBox(const Box&, std::optional<EscapeReason> = std::nullopt);
 
+    const InlineContentCache& inlineContentCache() const { return m_inlineContentCache; }
```

Comment:
Builders tend to communicate with each other by passing in objects as opposed to adding getters to generic public interfaces.

Reply:
Could you remove this please?